# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../grub/netboot-fixes.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- grub-0.97/netboot/ntulip.c.vanilla	2008-08-20 13:35:49.000000000 +0200
+++ grub-0.97/netboot/ntulip.c	2008-08-20 13:36:23.000000000 +0200
@@ -857,10 +857,8 @@
   /* initialize device */
   ntulip_reset(card);
 
-  card->reset    = ntulip_reset;
   card->poll     = ntulip_poll;
   card->transmit = ntulip_transmit;
-  card->disable  = ntulip_disable;
 
   return card;
 }
